script/functions.sh

```diff
@@ -307,33 +320,24 @@ function maven_install_libs() {
 
     func_echo "maven install libs ..."
 
-    for module_name in *; do
-        if [ -d "$module_name" ]; then
-            pushd "$module_name" || return 2
-            func_echo "start install lib $module_name"
-            if ! mvn clean install -Dmaven.test.skip=true ${maven_extra_args[@]}; then
-                func_echo "maven install lib $module_name with args ${maven_extra_args[@]} failed"
-            else
-                func_echo "maven install lib $module_name with args ${maven_extra_args[@]} succeed"
-            fi
-            popd
-        fi
-    done
+# 必须先打 ob-sql-parser 包
+    build_module "ob-sql-parser" "${maven_extra_args[@]}" || return 2
```

Comment:
may refer build_libs straightly, install_libs is for install outside jar file, not for build from odc source code

Reply:
I will try

Reply:
The original script was to iterate through the libs folder and then package it one by one, but since db-browser must be packaged later than ob-sql-parser, I think can only unpack the traversal and package it declaratively in order

Reply:
> The original script was to iterate through the libs folder and then package it one by one, but since db-browser must be packaged later than ob-sql-parser, I think can only unpack the traversal and package it declaratively in order

@yhilmare  has fixed this problem

Reply:
ok，but it has not merged into 43x branch